commands/handbook.py

```diff
@@ -0,0 +1,72 @@
+from commands.base import Command
+from discord import Embed
+from helpers import CommandFailure, is_good_response
+
+import bs4, re, requests
+
+
+def subject_details(code):
+    code = code.upper()
+
+    page = requests.get('https://www.handbook.unsw.edu.au/undergraduate/courses/2019/' + code)
+
+    if not is_good_response(page):
+        page = requests.get('https://www.handbook.unsw.edu.au/postgraduate/courses/2019/' + code)
+
+        if not is_good_response(page):
+            return None
+
+    soup = bs4.BeautifulSoup(page.text, features='lxml')
+
+    course_title = soup.find('span', attrs={'data-hbui' : 'module-title'}).string.strip()
+    course_offerings = soup.find('strong', string=re.compile('Offering Terms')).parent.contents[3].string.strip()
+
+    course_conditions_tag = soup.find('div', id='readMoreSubjectConditions')
+
+    course_conditions = ''
+    if course_conditions_tag is not None:
+        for s in course_conditions_tag.contents[1].contents[1].strings:
+            course_conditions += s
+    else:
+        course_conditions = 'None'
+
+    course_conditions = course_conditions.strip()
+
+    course_description_tag = soup.find('div', id='readMoreIntro')
+
+    course_description = ''
+
+    if len(course_description_tag.contents[1].contents) > 1 and course_description_tag.contents[1].contents[1].name == 'p':
+        for s in course_description_tag.contents[1].contents[1].strings:
+            course_description += s
+    else:
+        course_description = course_description_tag.contents[1].contents[0].string
```

Comment:
Same thing as previous conditional, make this the default value of course_description so that the `else` isn't needed